macros/alter_table_properties.sql

```diff
@@ -307,6 +307,15 @@ ALTER TABLE labels.all SET TBLPROPERTIES('dune.public'='true',
                                         'dune.data_explorer.contributors'='["soispoke","hildobby"]');
 {% endset %}
 
+{% set labels_arb_traders %}
+ALTER VIEW labels.nft SET TBLPROPERTIES('dune.public'='true',
```

Comment:
`ALTER VIEW labels.nft` needs to be changed into `ALTER TABLE labels.arb_traders` here